packages/mobile/src/transactions/ExchangeFeedItem.tsx

```diff
@@ -4,161 +4,44 @@ import colors from '@celo/react-components/styles/colors'
-  outValue: dollarAmount,
-  localCurrencyCode,
-  localCurrencySymbol,
-  localExchangeRate,
-}: ExchangeInputProps) {
-  const localAmount = getLocalAmount(dollarAmount, localExchangeRate)
-  return {
-    icon: require('src/transactions/ExchangeGoldGreen.png'),
-    dollarAmount,
-    dollarDirection: '',
-    localCurrencyCode,
-    localCurrencySymbol,
-    localAmount,
-    goldAmount,
-    goldDirection: '-',
-    inValue: goldAmount,
-    inColor: colors.celoGold,
-    outValue: localCurrencyCode ? localAmount : dollarAmount,
-    outColor: colors.celoGreen,
-  }
-}
-
-function getGoldAmountProps({ goldAmount: amount, goldDirection: amountDirection }: ExchangeProps) {
-  return {
-    amount,
-    amountDirection,
-    amountColor: colors.celoGold,
-    displayAmount: `${amountDirection}${getMoneyDisplayValue(amount)}`,
-  }
-}
-
-function getDollarAmountProps({
-  dollarAmount,
-  dollarDirection: amountDirection,
-  localCurrencyCode,
-  localCurrencySymbol,
-  localAmount,
-}: ExchangeProps) {
-  const amount = localCurrencyCode ? localAmount : dollarAmount
-  return {
-    amount,
-    amountDirection,
-    amountColor: colors.celoGreen,
-    displayAmount: amount
-      ? `${amountDirection}${localCurrencySymbol + getMoneyDisplayValue(amount)}`
-      : '-',
-  }
+type Props = ExchangeItemFragment & {
+  status: TransactionStatus
 }
 
+// TODO(jeanregisser): ExchangeFeedItem and TransferFeedItem renders are very similar, we should use the same building blocks
+// so the parts that need to be identical stay the same as we change the code (main layout)
 export function ExchangeFeedItem(props: Props) {
-  const { showGoldAmount, inSymbol, status, timestamp, t, i18n } = props
-
-  const localCurrencyCode = useLocalCurrencyCode()
-  const localCurrencySymbol = useLocalCurrencySymbol()
-  const localExchangeRate = useExchangeRate()
+  const { t, i18n } = useTranslation(Namespaces.walletFlow5)
+  const { amount, makerAmount, takerAmount, status, timestamp } = props
 
   const onPress = () => {
     navigateToExchangeReview(timestamp, {
-      makerToken: resolveCurrency(inSymbol),
-      makerAmount: new BigNumber(props.inValue),
-      takerAmount: new BigNumber(props.outValue),
+      makerAmount,
+      takerAmount,
     })
   }
 
-  const inCurrency = resolveCurrency(inSymbol)
-  const exchangeInputProps = { ...props, localCurrencyCode, localCurrencySymbol, localExchangeRate }
-  const exchangeProps =
-    inCurrency === CURRENCY_ENUM.DOLLAR
-      ? getDollarExchangeProps(exchangeInputProps)
-      : getGoldExchangeProps(exchangeInputProps)
-  const amountProps = showGoldAmount
-    ? getGoldAmountProps(exchangeProps)
-    : getDollarAmountProps(exchangeProps)
-
-  const { inValue, outValue } = exchangeProps
-  const { displayAmount, amountDirection, amountColor } = amountProps
-
+  const icon =
+    makerAmount.currencyCode === CURRENCIES[CURRENCY_ENUM.GOLD].code
+      ? require('src/transactions/ExchangeGoldGreen.png')
```

Comment:
please move these `require` calls to src/images/Images.ts and then use the exported values here